modules/metrics/src/test/java/org/apache/ignite/internal/metrics/ThreadPoolMetricTest.java

```diff
@@ -54,8 +54,9 @@ public void test() throws ExecutionException, InterruptedException {
 
         assertEquals(0L, completedTaskCount.value());
 
-        exec.submit(() -> {}).get();
-        assertEquals(1L, completedTaskCount.value());
+        exec.submit(() -> {}).get(1, TimeUnit.SECONDS);
+
+        waitForCondition(() -> completedTaskCount.value() > 0, 10, TimeUnit.SECONDS.toMillis(1));
```

Comment:
There should probable be an `assertTrue` here

Reply:
Also, the code in the end should be placed in either `finally` block, or `afterEach` block

Reply:
Fix it. thanks

Reply:
Fix it all